environments/hmpps-domain-services.json

```diff
@@ -0,0 +1,49 @@
+{
+  "account-type": "member",
+  "environments": [
+    {
+      "name": "development",
+      "access": [
+        {
+          "github_slug": "studio-webops",
+          "level": "sandbox"
+        }
+      ]
+    },
+    {
+      "name": "test",
+      "access": [
+        {
+          "github_slug": "studio-webops",
+          "level": "developer"
+        }
+      ]
+    },
+    {
+      "name": "preproduction",
+      "access": [
+        {
+          "github_slug": "studio-webops",
+          "level": "developer"
+        }
+      ]
+    },
+    {
+      "name": "production",
+      "access": [
+        {
+          "github_slug": "studio-webops",
+          "level": "developer"
+        }
+      ]
+    }
+  ],
+  "tags": {
+    "application": "hmpps-domain-services",
+    "business-unit": "HMPPS",
+    "infrastructure-support": "digital-studio-operations-team@digital.justice.gov.uk",
+    "owner": "sean.privett@digital.justice.gov.uk"
+  },
+  "go-live-date": "",
+  "github-oidc-team-name": "github-oidc-repositories"
```

Comment:
Copied base from `example.json`, not sure what to do with these two fields: `go-live-date` and `github-oidc-team-name` 🤔

Reply:
Sorry, thats my work currently , the "github-oidc-team-name" can be taken out